test/unit/ManagerProxy.js

```diff
@@ -1,35 +1,39 @@
 import Fixture from "./helpers/Fixture"
-import expectThrow from "../helpers/expectThrow"
 import {contractId} from "../../utils/helpers"
 import {toBuffer, bufferToHex} from "ethereumjs-util"
 import ethAbi from "ethereumjs-abi"
 
-const ManagerProxy = artifacts.require("ManagerProxy")
-const ManagerProxyTargetMockV1 = artifacts.require("ManagerProxyTargetMockV1")
-const ManagerProxyTargetMockV2 = artifacts.require("ManagerProxyTargetMockV2")
-const ManagerProxyTargetMockV3 = artifacts.require("ManagerProxyTargetMockV3")
+import {web3, ethers} from "hardhat"
 
-describe("ManagerProxy", accounts => {
+import chai, {expect, assert} from "chai"
+import {solidity} from "ethereum-waffle"
+chai.use(solidity)
+
+describe("ManagerProxy", () => {
     let fixture
     let managerProxy
+    let managerProxyFac
+    let signers
 
     describe("constructor", () => {
```

Comment:
can we move this `describe` statement after the `before`, `beforeEach` and `after` hooks, for better code readability? I got a bit confused with the `managerProxyFac` variable about its declaration and assignment